test/hotspot/gtest/gc/shared/test_bufferNodeAllocator.cpp

```diff
@@ -185,25 +185,30 @@ class BufferNode::TestSupport::ProcessorThread : public JavaTestThread {
 };
 
 static void run_test(BufferNode::Allocator* allocator, CompletedList* cbl) {
-  const uint nthreads = 4;
+
+  // deallocation is slower than allocation, so lets create
+  // more deallocation threads to prevent too large buildup of
+  // free nodes (footprint)
+  const uint nthreads_mut = 4;
+  const uint nthreads_proc = 6;
```

Comment:
Naming them "num_processor_threads" and "num_allocator_threads" is more consistent with the corresponding types `ProcessorThread` and `AllocatorThread`, IMO.